src/Components/Carousel/Carousel.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import { getImageInfo } from "../../Services/ongInfoServices";
+import { Swiper, SwiperSlide } from "swiper/react";
+import SwiperCore, {
+  Autoplay,
+  Navigation,
+  Pagination,
+  Scrollbar,
+  A11y,
+  Parallax,
+} from "swiper";
+import "swiper/swiper-bundle.min.css";
+import "swiper/swiper.min.css";
+import Styles from "./Carousel.module.css";
```

Comment:
the styles should be on the styles folder separated by component type: container, buttons, etc.

Reply:
changes added